azero/contracts/ownable/.gitignore

```diff
@@ -0,0 +1,9 @@
+# Ignore build artifacts from the local tests sub-crate.
+/target/
+
+# Ignore backup files creates by cargo fmt.
+**/*.rs.bk
```

Comment:
Question: can't this be covered more generally in the overall `.gitignore` for the repo?